graphite_api/functions.py

```diff
@@ -2482,14 +2672,43 @@ def timeShift(requestContext, seriesList, timeShift, resetEnd=True):
         &target=timeShift(Sales.widgets.largeBlue,"+1h")
 
     """
-    from .app import evaluateTarget, pathsFromTarget
     # Default to negative. parseTimeOffset defaults to +
     if timeShift[0].isdigit():
         timeShift = '-' + timeShift
     delta = parseTimeOffset(timeShift)
     myContext = requestContext.copy()
     myContext['startTime'] = requestContext['startTime'] + delta
     myContext['endTime'] = requestContext['endTime'] + delta
+
+    if alignDST:
+        def localDST(dt):
```

Comment:
Could that be a module-level utility function instead of a closure? It doesn't use anything in the local context.